### DISCUSSION TITLE
Is it possible to pause the waveform selection, autoscrolling when the user is scrolling through the waveform

### DISCUSSION THREAD

**Discussion by chandarsunderraj** [katspaugh/wavesurfer.js#3912] [Q&A]
It would be great if there is a timer option in milliseconds, once defined the waveform highlighting and auto scroll event can be paused for the defined seconds when the user is scrolling through the waveform enabling the user to choose a specific time duration by clicking or if the user is not scrolling for the defined seconds the autoscroll and waveform highlighting can continue.

**Comment by katspaugh** [comment]
Great idea! Wanna take a stab at it?
